active/RadioMenuFlyoutItem/RadioMenuFlyoutItem.md

```diff
@@ -0,0 +1,65 @@
+# Background
+A [MenuFlyoutItem](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.MenuFlyoutItem) is an item
+in a menu, be it a context menu ([MenuFlyout](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.MenuFlyout))
+or a [MenuBarItem](https://docs.microsoft.com/uwp/api/Windows.UI.Xaml.Controls.MenuBarItem).
+
+The new RadioMenuFlyoutItem in this spec is a MenuFlyoutItem that displays and behaves like a radio button
+(can be checked/unchecked, and in a group only one is selected at a time).
+
+# Description
+A radio menu flyout item is a menu item that is mutually exclusive with other radio menu items in its group. 
+This control contol allows you to present users with menus containing multiple options, where a user would choose only one of these options at a given time.
+
+
+# Examples
+
+## Create RadioMenuFlyoutItems
+*RadioMenuFlyoutItem* can be added into a *MenuBarItem*, *MenuFlyout*, or *MenuFlyoutSubItem*. The following example shows three radio menu flyout items as the content of a cascading menu flyout.
+
+![Three radio menu flyout items in a View goup that allow a user to select the size of icons](RadioMenuFlyoutItems.png)
+
+````Xaml
+<MenuFlyoutSubItem Text="View">
```

Comment:
Nit: could you put this into a <MenuFlyout>?  It's not obvious to someone not familiar with menus.

Reply:
I put it inside a MenuFlyout. I can go further and add the MenuFlyout to my Grid's ContextFlyout property, but that feels like it's taking the user's attention away from the real subject of the sample. So I'm keeping the sample really simple.